Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -39,6 +41,11 @@ static std::thread s_prefetcher;
 
 static const std::string s_format_prefix = "tex1_";
 
+HiresTexture::Level::Level()
+	: data(nullptr, SOIL_free_image_data)
```

Comment:
Will SOIL_free_image_data be called at all if data is nullptr? Else the default could be nullptr, nullptr.